### PR TITLE
Add proper JSON:API fields to run tasks example output

### PR DESCRIPTION
**Pull request by jarrodn-au**
### What
Updated 
* website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx
* website/public/schema/run-tasks/example-payload.json

### Why
Fixes for accuracy and edits for clarity around the use of JSON:API structures.

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [X] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [X] (N/A) Description links to related pull requests or issues, if any.

#### Content
- [X] (N/A) Redirects have been added to `website/redirects.js` for moved, renamed, or deleted pages.
- [X] API documentation and the API Changelog have been updated. 
- [X] (N/A) Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [X] (N/A)Pages with related content are updated and link to this content when appropriate.
- [X] (N/A)Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [X] (N/A)New pages have metadata (page name and description) at the top.
- [X] (N/A)New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [X] (N/A) New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [X] (N/A) UI elements (button names, page names, etc.) are bolded.
- [X] (N/A) The Vercel website preview successfully deployed.

#### Reviews
- [X] (N/A) I or someone else reviewed the content for technical accuracy.
- [X] (N/A) I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx [hunk 1]
```diff
@@ -180,7 +180,7 @@ Status values other than passed, failed, or running return an error. Both the pa
 
 #### Outcomes Payload Body
 
-A run task result may optionally contain one or more detailed outcomes, which improves result visibility and content in the Terraform Cloud user interface.
+A run task result may optionally contain one or more detailed outcomes, which improves result visibility and content in the Terraform Cloud user interface. In addition to the usual JSON:API fields (see the payload example below), the following attributes define the outcome.
```
Comment (glennsarti) [inline website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx:183]: >  usual JSON:API fields

I don't think we ever really define what are "usual JSON:API fields".  Are you just talking about `type` and `attributes` ?

Comment (jarrodn-au) [reply website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx:183]: Yeah, I was unsure how to wrap that up in a simple statement without adding (polluting?) the table with 'data' and 'attributes'. Perhaps

> In addition to the JSON:API fields of `type` and `attributes` (see the payload example below), the following properties define the outcome

Comment (glennsarti) [reply website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx:183]: Given it's inside the `attributes` object, I think we don't need that sentence.

```
A run task result may optionally contain one or more detailed outcomes, which improves result visibility and content in the Terraform Cloud user interface. The following attributes define the outcome.
```

Comment (jarrodn-au) [reply website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx]: Good call 👍

**Pull request merged**